src/restful/request_types/shared.py

```diff
@@ -36,10 +36,11 @@ class EntityUUID(BaseModel):
     uid: UUID4 = Field(..., alias="_id")
 
 
-class Reference(Entity, EntityName, EntityUUID):
-    @root_validator(pre=True)
-    def from_underscore_id_to_uid(cls, values):
-        return {**values, "uid": values.get("_id")}
+class Reference(BaseModel):
+    ref: str
+    type: common_type_constrained_string  # type: ignore
+    targetType: Optional[common_type_constrained_string]  # type: ignore
+    targetName: Optional[common_name_constrained_string]  # type: ignore
```

Comment:
```suggestion
    targetType: common_type_constrained_string | None = None  # type: ignore
    targetName: common_name_constrained_string| None = None # type: ignore
```
New recommended syntax. "No implisit default None"